OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java

```diff
@@ -36,12 +37,11 @@ void completePlayerSetup(boolean asked) {
 			skinLayout = (OoyalaSkinLayout) findViewById(R.id.ooyalaSkin);
 			// Create the OoyalaPlayer, with some built-in UI disabled
 			PlayerDomain playerDomain = new PlayerDomain(domain);
-			Options options = null;
-			if(selectedFormat.equalsIgnoreCase("default"))
-				options = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).build();
-			else
-				options = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).setPlayerInfo(new CustomPlayerInfo(selectedFormat)).build();
-			player = new OoyalaPlayer(pcode, playerDomain, options);
+			player = new OoyalaPlayer(pcode, playerDomain, getOptions());
+			if(isStaging) {
```

Comment:
What about the else option in case you don't want to use staging but production? Staging will continue selected after setting Staging once even if you deselect the option.

Reply:
Added else option as well